airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/SshPostgresDestinationAcceptanceTest.java

```diff
@@ -163,7 +164,7 @@ private static Database getDatabaseFromConfig(final JsonNode config) {
   @Override
   protected void setup(final TestDestinationEnv testEnv) throws Exception {
     // do everything in a randomly generated schema so that we can wipe it out at the end.
-    schemaName = RandomStringUtils.randomAlphanumeric(5);
+    schemaName = RandomStringUtils.randomAlphabetic(8).toLowerCase();
```

Comment:
changed this because:
hitting errors when schema name has numeric as first character
had issue for dbt seed on case mismatch